mezzrow/oscar_apps/checkout/views.py

```diff
@@ -10,13 +10,22 @@
 
 BankcardForm = get_class('payment.forms', 'BankcardForm')
 
+from smallslive.users.models import SmallsUser
+
 
 class IndexView(views.IndexView):
     def form_valid(self, form):
         if form.is_guest_checkout():
             first_name = form.cleaned_data['first_name']
             last_name = form.cleaned_data['last_name']
             self.checkout_session.set_reservation_name(first_name, last_name)
+
+            # Capture this email address if it's not already in the database
+            a = SmallsUser.objects.get_or_create(
```

Comment:
Can't remember if Mezzrow already allows non-guest checkouts, but if it does, won't adding a new user make it impossible for people to do non-guest checkout in the future since a user with that email is already created?